docs/fr/guides/utiliser-des-polices-d'écriture-personnalisées

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
Are you able to delete that file? It got duplicated because the one on main has the `.md` extension it seems... I tried to delete it but it's not letting me

Reply:
You're the owner it seems, I can't touch it
I'll try deleting it in my fork and making a PR

Reply:
> You're the owner it seems, I can't touch it I'll try deleting it in my fork and making a PR

I already fixed and merged the PR so it should be fine